### PR TITLE
Remove mention of Istio managed certmanager

### PR DESCRIPTION
**Pull request by howardjohn**
Fixes https://github.com/istio/istio.io/issues/5022

### CODE REVIEW

File: content/en/docs/tasks/traffic-management/ingress/ingress-certmgr/index.md [hunk 1]
```diff
@@ -22,15 +22,15 @@ $ helm template $HOME/istio-fetch/istio \
...
-  --set certmanager.enabled=true \
-  --set certmanager.email=mailbox@donotuseexample.com \
   > $HOME/istio-fetch/istio.yaml
 {{< /text >}}
 
 {{< tip >}}
 By default `istio-ingressgateway` will be exposed as a `LoadBalancer` service type. You may want to change that by setting the `gateways.istio-ingressgateway.type` installation option to `NodePort` if this is more applicable to your Kubernetes environment.
 {{< /tip >}}
 
+[Install cert-manager](https://docs.cert-manager.io/en/latest/getting-started/install/kubernetes.html) if you want to manage certificates automatically.
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/ingress/ingress-certmgr/index.md]: Conditionally installing cert-manger "if you ..." sounds wrong given the title "Kubernetes Ingress with Cert-Manager".

Comment (frankbu) [reply content/en/docs/tasks/traffic-management/ingress/ingress-certmgr/index.md]: Also, how about making the "Install Istio" and "Install cert-manager" two steps in a numbered list?

File: content/en/docs/tasks/traffic-management/ingress/ingress-certmgr/index.md [hunk 2]
```diff
@@ -22,15 +22,15 @@ $ helm template $HOME/istio-fetch/istio \
...
   --set global.k8sIngress.enableHttps=true \
   --set global.k8sIngress.gatewayName=ingressgateway \
-  --set certmanager.enabled=true \
-  --set certmanager.email=mailbox@donotuseexample.com \
   > $HOME/istio-fetch/istio.yaml
 {{< /text >}}
 
 {{< tip >}}
 By default `istio-ingressgateway` will be exposed as a `LoadBalancer` service type. You may want to change that by setting the `gateways.istio-ingressgateway.type` installation option to `NodePort` if this is more applicable to your Kubernetes environment.
 {{< /tip >}}
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/ingress/ingress-certmgr/index.md]: Lines 28-30 need to be indented 4 spaces for the numbered list to work.

File: content/en/docs/tasks/traffic-management/ingress/ingress-certmgr/index.md [hunk 3]
```diff
@@ -22,15 +22,15 @@ $ helm template $HOME/istio-fetch/istio \
   --set global.k8sIngress.enabled=true \
   --set global.k8sIngress.enableHttps=true \
   --set global.k8sIngress.gatewayName=ingressgateway \
-  --set certmanager.enabled=true \
-  --set certmanager.email=mailbox@donotuseexample.com \
   > $HOME/istio-fetch/istio.yaml
 {{< /text >}}
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/ingress/ingress-certmgr/index.md]: Also, lines 18-26 need to be indented 4 spaces for the numbered list to work.

Comment (howardjohn) [reply content/en/docs/tasks/traffic-management/ingress/ingress-certmgr/index.md]: Thanks! verified with `make serve` this works now

File: content/en/docs/tasks/traffic-management/ingress/ingress-certmgr/index.md [hunk 4]
```diff
@@ -13,23 +13,23 @@ You will start with a clean Istio installation, create an example service, expos
...
+      --set global.k8sIngress.enableHttps=true \
+      --set global.k8sIngress.gatewayName=ingressgateway \
+      > $HOME/istio-fetch/istio.yaml
+    {{< /text >}}
+
+    {{< tip >}}
+    By default `istio-ingressgateway` will be exposed as a `LoadBalancer` service type. You may want to change that by setting the `gateways.istio-ingressgateway.type` installation option to `NodePort` if this is more applicable to your Kubernetes environment.
+    {{< /tip >}}
+
+1. [Install cert-manager](https://docs.cert-manager.io/en/latest/getting-started/install/kubernetes.html).
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/ingress/ingress-certmgr/index.md]: Just one more nit :)
How about making this the same as in the other task?
```suggestion
1. [Install cert-manager](https://docs.cert-manager.io/en/latest/getting-started/install/kubernetes.html) to manage certificates automatically.
```

**Pull request merged**